sdk/nodejs/api/client.ts

```diff
@@ -46,6 +51,7 @@ class BaseClient {
 
 
 
+
```

Comment:
There's quite a lot of whitespace in these, I assume it's because of go templates?

Reply:
Yes, I can improve this.

Reply:
I guess it's good now.